lib/siwapp/customers/customer.ex

```diff
@@ -66,4 +71,22 @@ defmodule Siwapp.Customers.Customer do
       add_error(changeset, hd(fields), "Either name or identification are required")
     end
   end
+
+  defp create_hash_id(changeset) do
+    name =
+      changeset
+      |> get_field(:name) ||
+        ""
+        |> String.downcase()
+        |> String.replace(~r/ +/, "")
+
+    identification =
+      changeset
+      |> get_field(:identification) ||
+        ""
```

Comment:
I think this could be better defining a new fn like:
```elixir
defp get_field_or_empty(field) do
  get_field(field) || ""
end
```
and using that in the pipes